tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/mqtt/imported/MQTTTestSupport.java

```diff
@@ -57,6 +57,11 @@
 import org.apache.activemq.artemis.spi.core.remoting.Acceptor;
 import org.apache.activemq.artemis.spi.core.security.ActiveMQJAASSecurityManager;
 import org.apache.activemq.artemis.tests.util.ActiveMQTestBase;
+import org.fusesource.hawtdispatch.DispatchPriority;
```

Comment:
Is this fusesource actually maintained, i see last update 2016, do we really want to be building further on it???

Can we implement using alternative lib or our own for better maintenance?

Reply:
FuseMQTTClientProvider (already used by some MQTT tests) has a transitive dependency for org.fusesource.hawtdispatch: HawtDispatcher is statically allocated by DispatcherConfig when org.fusesource.mqtt.client.MQTT.blockingConnection is called and It create a new GlobalDispatchQueue.
I used org.fusesource.hawtdispatch only to shuthdown the global queue at the end of the test to avoid file descriptor leaks.